src/utils/deploy.ts

```diff
@@ -75,6 +88,9 @@ export async function executeDeploy(
     },
   });
 
+  const cache = await DeployCache.create();
+  cache.set(deploy.id, { ...opts, wait: opts.wait.quantity });
```

Comment:
opts.wait.minutes, in case someone (ex, UT) is passing in some other duration unit?